packages/table/src/spectrum-table-body.css

```diff
@@ -14,16 +14,13 @@ governing permissions and limitations under the License.
 :host {
     border-radius: var(--mod-table-border-radius, var(--spectrum-table-border-radius));
     border: none;
+    display: table-row-group;
     position: relative;
 }
 
 :host([drop-target]) {
-    --spectrum-table-border-color: transparent;
+    --mod-table-border-color: transparent;
```

Comment:
We shouldn't override the `mod` property here, it might break consumers?

Reply:
It's specific to the drop-target context which doesn't support a border color --mod override. We wouldn't want the --mod-table-border-color to show up no matter what value was set in the drop-target context because it would make the interface look broken. So the tldr; here is that this is a design direction _not_ to allow the border-color show at all in this state.